src/builtin-projections/defaults.js

```diff
@@ -10,60 +9,54 @@ function normalize(config) {
     tfoot = {},
   } = config;
 
-  const colgroup = _.extend({ columns }, config.colgroup);
-  const tbody = _.extend({ records }, config.tbody);
-  const colgroups = _.chain(config)
-    .result('colgroups', [{ key: 'colgroup-default' }])
-    .map(cg => (cg.key === 'colgroup-default' ? _.defaults(colgroup, cg) : cg))
-    .value();
-  const tbodies = _.chain(config)
-    .result('tbodies', [{ key: 'tbody-default' }])
-    .map(tb => (tb.key === 'tbody-default' ? _.defaults(tbody, tb) : tb))
-    .value();
-
-  return _.defaults({
+  const colgroup = Object.assign({}, { columns }, config.colgroup);
+  const tbody = Object.assign({}, { records }, config.tbody);
+  const colgroups = (config.colgroups || [{ key: 'colgroup-default' }]).map(cg =>
+    (cg.key === 'colgroup-default' ? Object.assign(cg, colgroup) : cg));
+  const tbodies = (config.tbodies || [{ key: 'tbody-default' }]).map(tb =>
+    (tb.key === 'tbody-default' ? Object.assign(tb, tbody) : tb));
+
+  return Object.assign({}, config, {
```

Comment:
`Object.assign` is not supported in IE, so we expect user to have polyfill?

Reply:
IE sucks!!!!!!!!!!!